README.md

```diff
@@ -301,6 +300,30 @@ with this option enabled.
          "user": "flexion"
          }
 
+### Useful Scipts
+
+Reset script - Fix the filepaths before using.
```

Comment:
Thank you for adding this! I was thinking we could put the script in a /scripts folder at the top level of the repo